### PR TITLE
Update virus-detection-in-spo.md

### PR DESCRIPTION
**Pull request by AdwoodMicrosoft**
clarity around the AV scan order of events and asynchronous wording.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/virus-detection-in-spo.md [hunk 1]
```diff
@@ -36,16 +36,16 @@ Microsoft 365 uses a common virus detection engine for scanning files that users
 > [!IMPORTANT]
 > The built-in anti-virus capabilities are a way to help contain viruses. They aren't intended as a single point of defense against malware for your environment. We encourage all customers to investigate and implement anti-malware protection at various layers and apply best practices for securing their enterprise infrastructure. For more information about strategies and best practices, see [Security roadmap](security-roadmap.md).
 
-## What happens when an infected file is uploaded to SharePoint Online?
+## What happens if an infected file is uploaded to SharePoint Online?
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/virus-detection-in-spo.md:39]:
```suggestion
## What happens if an infected file is uploaded to SharePoint Online?
```

File: microsoft-365/security/office-365-security/virus-detection-in-spo.md [hunk 2]
```diff
@@ -36,16 +36,16 @@ Microsoft 365 uses a common virus detection engine for scanning files that users
...
 
-The Microsoft 365 virus detection engine runs asynchronously within SharePoint Online. **All files are not automatically scanned on upload**. Heuristics determine the files to scan. When a file is found to contain a virus, the file is flagged so it can't be downloaded again. In April 2018, we removed the 25 MB limit for scanned files.
+The Microsoft 365 virus detection engine runs asynchronously (idenpedent from file uploads) within SharePoint Online. **All files are not automatically scanned**. Heuristics determine the files to scan. When a file is found to contain a virus, the file is flagged. In April 2018, we removed the 25 MB limit for scanned files.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/virus-detection-in-spo.md]:
```suggestion
The Microsoft 365 virus detection engine runs asynchronously (independent from file uploads) within SharePoint Online. **All files are not automatically scanned**. Heuristics determine the files to scan. When a file is found to contain a virus, the file is flagged. In April 2018, we removed the 25 MB limit for scanned files.
```

File: microsoft-365/security/office-365-security/virus-detection-in-spo.md [hunk 3]
```diff
@@ -36,16 +36,16 @@ Microsoft 365 uses a common virus detection engine for scanning files that users
...
 
 Here's what happens:
 
 1. A user uploads a file to SharePoint Online.
-2. SharePoint Online determines whether the file meets the criteria for a scan.
-3. The virus detection engine scans the file.
-4. If a virus is found, the virus engine sets a property on the file indicating that it's infected.
+2. SharePoint Online as part of its virus scanning processes later determines if the file meets the criteria for a scan.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/virus-detection-in-spo.md]:
```suggestion
2. SharePoint Online, as part of its virus scanning processes, later determines if the file meets the criteria for a scan.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@AdwoodMicrosoft

**Pull request merged**